robottelo/decorators.py

```diff
@@ -40,6 +40,32 @@
 }
 
 
+def skip_if_not_set(*options):
+    """Skips test if expected configuration is not set.
+
+    :param options: List of valid `robottelo.properties `section names.
```

Comment:
I think it will be great to validate the options otherwise it will be difficult to find typo errors - possible values should be one of this list:
```py
 ['server', 'robottelo', 'ldap', 'fake_manifest', 'clients', 'docker', 'rhai', 'transition', 'performance', 'compute_resources']
```